EZEE-2772 Calendar widget (#38)
ezsystems/user-documentation: docs/publishing/advanced_publishing_options.md

    Out of the box, it displays Content items scheduled for future publication, but your page administator can configure custom events.
    Therefore, it can contain other events, e.g. national holidays, important dates, etc.
    
    To access the calendar widget, in the **Content Panel**, open the **Calendar** tab.
    
    ![Calendar widget](img/calendar_widget.png "Calendar widget")
    
    You can switch between different views and access filters.
    
    |Button|Description|
    |------|-----------|
    |![Today button](img/calendar_widget_today.png)|Display the current day regardless of the view.|
    |![Arrows](img/calendar_widget_arrows.png)|Navigate through months, weeks, or days.|
    |![Toggle month/week/day](img/calendar_widget_toggler_mwd.png)|Switch between the month, week, or day view.|
    |![Toggle calendar/list](img/calendar_widget_toggler_cal_list.png)|Switch between the calendar and list view,|
    |![Filters](img/calendar_widget_filters.png)|Access and apply calendar filters.|
    
    ### Calendar filters
        
    With numerous events appearing in the calendar widget, you can declutter the view by applying filters.

>     You can filter by the events' type or modified language in all three — month, week, and day — views.

Reviewer: ```suggestion
    You can filter by the events' type or modified language in all three views (month, week, and day).
```

    
    The calendar widget toolbar displays events you select and gives you access to actions assigned to them.
    The available actions appear in the upper-right corner of the toolbar when you select an event.
    You can select multiple events of the same type and perform bulk actions on them, e.g. [reschedule or cancel publication](#rescheduling-or-cancelling-events).
    
    !!! note
            
        Actions available in the toolbar may vary depending on the custom configuration.
        For details, contact your page administrator.
                
    
    To select, click on all events you want to add to the toolbar list.
    
    ![Calendar widget toolbar](img/selection_action_bar.png "Calendar widget toolbar")
    
    #### Rescheduling or cancelling events
    
    In case of publishable events (e.g. articles), you can change or cancel their planned publication by using the toolbar buttons.
    
    |Button|Description|
    |------|-----------|
    |![Reschedule button](img/selection_action_bar_reschedule.png)|Reschedule all selected events.|
    |![Discard button](img/selection_action_bar_discard.png)|Cancel the future publication of selected events.|
    

>     Select all events to perform a bulk action on them.

Reviewer: ```suggestion
    Select all events you want to perform a bulk action on.
```

    Using the toolbar buttons, perform one of the following actions:
    
    - To change the publication time, click the **Reschedule** button.

>     In the **Reschedule** content modal window, select the new date and click **Confirm reschedule**.

Reviewer: ```suggestion
    In the **Reschedule** modal window, select the new date and click **Confirm reschedule**.
```
Unless "Reschedule content" is the full button name?
